model/src/test/java/uk/gov/pay/commons/api/json/ApiResponseDateTimeDeserializerTest.java

```diff
@@ -42,11 +43,12 @@ public void shouldDeserializeToNull() throws IOException {
         assertNull(response.getCreatedDate());
     }
 
-    @Test(expected = JsonMappingException.class)
-    public void shouldThrowExceptionWhenNullValue() throws IOException {
+    @Test
+    public void shouldThrowExceptionWhenNullValue() {
         String testValue = "{\"created_date\":\"blah\"}";
-        final RequestDateTimeJsonTest response = objectMapper.readValue(testValue, RequestDateTimeJsonTest.class);
-        assertNull(response.getCreatedDate());
```

Comment:
I assume it never got as far as this assertion before?

Reply:
yeah, it never did